packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/variable-desc.md

```diff
@@ -1,3 +1,13 @@
-For example, if you have an OIDC configuration variable called `subdomain`, then you can set your redirect URL string to `https://{app.subdomain}.example.org/strawberry/login`. When your customer admin sets their `subdomain` variable value to `berryfarm`, then `https://berryfarm.example.org/strawberry/login` is their redirect URL.
+For example, if you have an OIDC configuration variable called `subdomain`, then you can set your **Redirect URI** string to `https://{app.subdomain}.example.org/strawberry/login`. When your customer admin sets their `subdomain` variable value to `berryfarm`, then `https://berryfarm.example.org/strawberry/login` is their redirect URL.
```

Comment:
"Customer admin" sounds awkward. Can we revise?

Reply:
I'll update this and remove "admin".